docs/topics/forms/formsets.txt

```diff
@@ -775,9 +775,79 @@ But with ``ArticleFormset(prefix='article')`` that becomes:
 This is useful if you want to :ref:`use more than one formset in a view
 <multiple-formsets-in-view>`.
 
+.. _formset-rendering:
+
 Using a formset in views and templates
 ======================================
 
+Formsets have five attributes and five methods associated with rendering.
+
+.. attribute:: BaseFormSet.renderer
+
+    Specifies the :doc:`renderer </ref/forms/renderers>` to use for the 
+    formset. Defaults to the renderer specified by the :setting:`FORM_RENDERER`
+    setting.
+
+.. attribute:: BaseFormSet.template_name
+
+    The name of the template used when calling ``__str__`` or 
+    :meth:`.render()`. This template renders the formsets management
+    forms and then each form in the formset as per the template defined by the
+    forms :attr:`~django.forms.Form.template_name`. This is a proxy of 
+    ``as_table`` by default.
+
+.. attribute:: BaseFormSet.template_name_p
+
+    The name of the template used when calling :meth:`.as_p()`. By default
+    this is ``'django/forms/formsets/p.html'``. This template renders the
+    formsets management forms and then each form in the formset as per the 
+    forms :meth:`~django.forms.Form.as_p()` method.
+
+.. attribute:: BaseFormSet.template_name_ul
+
+    The name of the template used when calling :meth:`.as_ul()`. By default
+    this is ``'django/forms/formsets/ul.html'``. This template renders the
+    formsets management forms and then each form in the formset as per the 
+    forms :meth:`~django.forms.Form.as_ul()` method.
```

Comment:
Do we need to add these? We don't currently support this so it's a new feature right? 🤔


I guess I'm thinking `render()` and `template_name` might be enough here?